broker/src/main/java/org/apache/rocketmq/broker/processor/SendMessageProcessor.java

```diff
@@ -351,6 +351,21 @@ private RemotingCommand handlePutMessageResult(PutMessageResult putMessageResult
                 response.setRemark(String.format("the message is illegal, maybe msg body or properties length not matched. msg body length limit %dB, msg properties length limit 32KB.",
                     this.brokerController.getMessageStoreConfig().getMaxMessageSize()));
                 break;
+            case WHEEL_TIMER_MSG_ILLEGAL:
+                response.setCode(ResponseCode.WHEEL_TIMER_MSG_ILLEGAL);
+                response.setRemark(String.format("the accurate delay message is illegal, delay time max limit %dms, or check props if setting TIMER_DEL_UNIQKEY or not",
+                        this.brokerController.getMessageStoreConfig().getTimerMaxDelaySec() * 1000));
+                break;
+            case WHEEL_TIMER_FLOW_CONTROL:
+                response.setCode(ResponseCode.WHEEL_TIMER_FLOW_CONTROL);
+                response.setRemark(String.format("or send message too much, triggered the flow control max num limit is %d or the current value is greater than %d and less than %d, trigger random flow control",
+                        this.brokerController.getMessageStoreConfig().getTimerCongestNumEachSlot() * 2L, this.brokerController.getMessageStoreConfig().getTimerCongestNumEachSlot(), this.brokerController.getMessageStoreConfig().getTimerCongestNumEachSlot() * 2L));
+                break;
+            case WHEEL_TIMER_NOT_ENABLE:
+                response.setCode(ResponseCode.WHEEL_TIMER_NOT_ENABLE);
+                response.setRemark(String.format("accurate delay is not enabled, timerWheelEnable is %s",
+                        brokerController.getMessageStoreConfig().isTimerWheelEnable()));
```

Comment:
How about reusing ResponseCode.SYSTEM_ERROR for WHEEL_TIMER_FLOW_CONTROL and WHEEL_TIMER_NOT_ENABLE?